doc/host_config_schema/cchost_config.json

```diff
@@ -82,12 +82,18 @@
                   },
                   "acme_configuration": {
```

Comment:
Should this be `null` when the RPC interface doesn't have ACME endorsement?

Reply:
I found a way to do that in the schema! Also, full disclosure, every I time I look at a json schema, I hate them a little more.